### PR TITLE
Change some literals

### PR DESCRIPTION
**Pull request by tramuntanal**
- Change button labels at Courses card
- Change title "Banc de recursos" for "Recursos"

### CODE REVIEW

**Review by rbngzlv — COMMENTED** [review]
I've done a quick review @tramuntanal .

File: decidim-resource_banks/config/locales/es.yml [hunk 1]
```diff
@@ -26,87 +26,87 @@ es:
...
-          title: Nueva administradora del banco de recursos.
+          title: Nueva administradora del recurso.
         update:
-          error: Se ha producido un error al actualizar una administradora para este banco de recursos.
-          success: Administradora actualizada correctamente para este banco de recursos.
+          error: Se ha producido un error al actualizar una administradora para este recurso.
+          success: Administradora actualizada correctamente para este recurso.
       menu:
-        resource_banks: Bancos de recursos
+        resource_banks: Recurso
```
Comment (victorol1) [inline decidim-resource_banks/config/locales/es.yml:78]:
```suggestion
        resource_banks: Recursos
```

**Review by victorol1 — APPROVED** [review]
LGTM, we can merge after solving @rbngzlv suggestions, I've removed mine since they were a duplicate.

**Comment by tramuntanal** [comment]
Thank you @victorol1 and @rbngzlv !
I've just update the :es bundle with your corrections

**Review by rbngzlv — APPROVED** [review]

**Pull request merged**